src/View/mainpage/MainPage.py

```diff
@@ -485,3 +498,38 @@ def on_loaded_suv2roi(self):
 
         # Close progress window
         self.suv2roi_progress_window.close()
+
+    def add_draw_roi_instance(self):
+        """Use ROIDrawOption controller to add the roi instance to the main window"""
+        logging.debug("add_draw_roi_instance started")
+
+        self.splitter.setVisible(False)
+        self.draw_roi = self.roi_draw_handler.show_roi_draw_window()
+        self.main_content.addWidget(self.draw_roi)
+        self.draw_roi_toggle_toolbar_items(True)
+
+    def remove_draw_roi_instance(self):
+        """removes the draw roi instance from the main window, also removing the instance from the ROIDrawOption controller"""
+        logging.debug("remove_draw_roi_instance started")
+
+        self.roi_draw_handler.remove_roi_draw_window()
+        self.main_content.removeWidget(self.draw_roi)
+        self.draw_roi_toggle_toolbar_items(False)
+        self.splitter.setVisible(True)
+
+
+    def draw_roi_toggle_toolbar_items(self, disabled):
+        """Called to disable toolbar options when they do not apply / cannot be used in the current draw roi context"""
+        self.action_handler.action_save_structure.setDisabled(disabled)
+        self.action_handler.action_save_as_anonymous.setDisabled(disabled)
+
+        self.action_handler.action_zoom_out.setDisabled(disabled)
+        self.action_handler.action_zoom_in.setDisabled(disabled)
+        self.action_handler.menu_windowing.setDisabled(disabled)
+        self.action_handler.windowing_window.setDisabled(disabled)
```

Comment:
I cant get the windowing icon to disable for some reason. Currently a bug, however it could be avoided by actually implementing it